docs/API.md

```diff
@@ -214,7 +214,7 @@ to be dropped before sending (so -1 should be passed here).
 The `srctime` parameter is an SRT addition for applications (i.e. gateways)
 forwarding SRT streams. It permits pulling and pushing of the sender's original time
 stamp, converted to local time and drift adjusted. The srctime parameter is the
-number of usec (since epoch) in local time. If the connection is not between
+number of usec (since epoch) in local SRT clock time. If the connection is not between
```

Comment:
Don't forget to update this when added the functions to obtain current local SRT time.